merlin/systems/dag/runtimes/triton/runtime.py

```diff
@@ -30,17 +30,23 @@
 from merlin.core.protocols import Transformable  # noqa
 from merlin.dag import Graph  # noqa
 from merlin.systems.dag.ops import compute_dims  # noqa
+from merlin.systems.dag.ops.fil import PredictForest  # noqa
 from merlin.systems.dag.ops.operator import add_model_param  # noqa
 from merlin.systems.dag.runtimes import Runtime  # noqa
+from merlin.systems.dag.runtimes.triton.ops.fil import PredictForestTriton  # noqa
 from merlin.systems.dag.runtimes.triton.ops.tensorflow import PredictTensorflowTriton  # noqa
 
+# Should this have an import if statement? XGboost, sklearn, CUML???
```

Comment:
We might need to figure out a pattern for this, since I don't think we want to make running things in Triton require all the modeling frameworks

Reply:
Actually, it looks like we have the start of a pattern a few lines farther down 😃